crates/languages/src/css/config.toml

```diff
@@ -1,6 +1,6 @@
 name = "CSS"
 grammar = "css"
-path_suffixes = ["css"]
+path_suffixes = ["css", "postcss"]
```

Comment:
Do people actually use `.postcss` as a file extension?

I've only ever seen it used with `.css`.

Reply:
If we're just adding this to recognize `<style lang="postcss">` I think that would be better handled in the injections query itself.

Reply:
Yeah it is in various other plugins

https://marketplace.visualstudio.com/items?itemName=csstools.postcss

It might even be worth adding `pcss` but up to you.